src/client/datascience/notebook/notebookControllerManager.ts

```diff
@@ -195,7 +182,7 @@ export class NotebookControllerManager implements INotebookControllerManager, IE
                     traceInfo('Find preferred kernel cancelled');
                     return;
                 }
-                await this.createOrDeletePlaceholderPythonKernel();
```

Comment:
If I understand the issue correctly the id that the dummy kernel has needs to be updated in order for it to work after you reload VS code?

Perhaps this isn't true? Can we just use the same id for the dummy kernel forever? Save it in user memento or something.

Reply:
>  use the same id for the dummy kernel forever? Save it in user memento or something.

Yes agreed, we could (i didnt want to add that code here), if i did go down that path, I'd be  adding the solution for pre-loads where we have a dummy solution.

Reply:
I.e. i'd like to fix this problem once we have a better understanding of the problem & solution. Right now it seems like i'm going down the path of fixing (coming up with a solution) as we implement something.

Reply:
So you're backing out the idea of a dummy kernel entirely, and waiting to do this until we decide on how to handle preload kernels?

Reply:
Current implementation feels super reactionary to me, so I'm super pro backing off and approaching this with more of a plan.